src/BaGet.Core/Metadata/BaGetPackageMetadata.cs

```diff
@@ -24,5 +24,11 @@ public class BaGetPackageMetadata : PackageMetadata
 
         [JsonProperty("repositoryType")]
         public string RepositoryType { get; set; }
+
+        /// <summary>
+        /// The package's release notes.
+        /// </summary>
+        [JsonProperty("releaseNotes")]
+        public string ReleaseNotes { get; set; }
```

Comment:
Could you move this to after `PackageTypes` but before `RepositoryUrl` to keep these properties alphabetically sorted?